docs/schema-loader.md

```diff
@@ -182,139 +219,184 @@ Create/Delete JDBC schemas
+$ java -jar scalardb-schema-loader-<VERSION>.jar --jdbc -j <JDBC_URL> -u <USER> -p <PASSWORD> -f schema.json
 ```
 
 ### Alter tables
 
-This command will add new columns and create/delete secondary index for existing tables. It compares
-the provided table schema to the existing schema to decide which columns need to be added and which
-indexes need to be created or deleted.
+You can use a command to add new columns to and create or delete a secondary index for existing tables. This command compares the provided table schema to the existing schema to decide which columns need to be added and which indexes need to be created or deleted.
 
-For using config file (Sample config file can be found [here](https://github.com/scalar-labs/scalardb/blob/master/conf/database.properties)):
+To add new colums to and create or delete a secondary index for existing tables, run the following command, replacing `<VERSION>` with the version of ScalarDB that you're using:
 
 ```console
-$ java -jar scalardb-schema-loader-<version>.jar --config <PATH_TO_CONFIG_FILE> -f schema.json --alter
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config <PATH_TO_PROPERTIES_FILE> -f schema.json --alter
 ```
 
+{% capture notice--info %}
+**Note**
+
+For a sample configuration, see [`database.properties`](https://github.com/scalar-labs/scalardb/blob/master/conf/database.properties).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
 For using CLI arguments fully for configuration (Deprecated. Please use the command using a config
 file instead):
 
 ```console
 # For Cosmos DB for NoSQL
-$ java -jar scalardb-schema-loader-<version>.jar --cosmos -h <COSMOS_DB_FOR_NOSQL_ACCOUNT_URI> -p <COSMOS_DB_FOR_NOSQL_KEY> -f schema.json --alter
+$ java -jar scalardb-schema-loader-<VERSION>.jar --cosmos -h <COSMOS_DB_FOR_NOSQL_ACCOUNT_URI> -p <COSMOS_DB_FOR_NOSQL_KEY> -f schema.json --alter
 ```
 
 ```console
 # For DynamoDB
-$ java -jar scalardb-schema-loader-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f schema.json --alter
+$ java -jar scalardb-schema-loader-<VERSION>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f schema.json --alter
 ```
 
 ```console
 # For Cassandra
-$ java -jar scalardb-schema-loader-<version>.jar --cassandra -h <CASSANDRA_IP> [-P <CASSANDRA_PORT>] [-u <CASSANDRA_USER>] [-p <CASSANDRA_PASSWORD>] -f schema.json --alter
+$ java -jar scalardb-schema-loader-<VERSION>.jar --cassandra -h <CASSANDRA_IP> [-P <CASSANDRA_PORT>] [-u <CASSANDRA_USER>] [-p <CASSANDRA_PASSWORD>] -f schema.json --alter
 ```
 
 ```console
 # For a JDBC database
-$ java -jar scalardb-schema-loader-<version>.jar --jdbc -j <JDBC URL> -u <USER> -p <PASSWORD> -f schema.json --alter
+$ java -jar scalardb-schema-loader-<VERSION>.jar --jdbc -j <JDBC_URL> -u <USER> -p <PASSWORD> -f schema.json --alter
 ```
```

Comment:
Should we should keep these `Deprecated` parts for these databases in the supported version branches (`3.10` to `3.5`) and on the `master` branch? Or we should completely remove these from the doc in all or some branches?

> **Note**
> 
> I haven't revised these parts yet because I wanted to confirm if we need to keep them first.